src/components/index.ts

```diff
@@ -105,3 +105,8 @@ export * from './icons';
 export * from './layout';
 export * from './table';
 export * from './carousel';
+
+export {
```

Comment:
we might need the create tenantLabel util to format the username we pass into the header

Reply:
That's a fair point - I was thinking about it too. I think it'll end up not being necessary at all with the full implementation, as we show `Garage name, location`), and the package will take care of that. But I'll see if it's true once I get further along